chapter2/tests/mkcert/sites-available/camp-php.local.conf

```diff
@@ -0,0 +1,31 @@
+server {
+        if ($host = camp-php.local) {
+            return 301 https://$host$request_uri;
+        }
+
+	    server_name camp-php.local;
+	    listen 80;
+    	root /opt/camp/camp-php/;
+}  
+
+
+server {
+	    server_name camp-php.local;
+	    listen 443 ssl;
+	    ssl_certificate /opt/certs/camp-php.local+4.pem;
+    	ssl_certificate_key /opt/certs/camp-php.local+4-key.pem;
+    	access_log /var/log/nginx/access.log logger-json;
+
+
+        location / {
+	    root  /opt/camp/camp-php/;
+	    index  index.html;
+        }
+
+
+        location /hello {
+		default_type text/html;
+		return 200 $query_string;
+        }
```

Comment:
indentation you have is like this
![image](https://user-images.githubusercontent.com/70822710/231887776-bf7cea9c-5228-4dd3-9b36-c44e59d1b398.png)